.github/tests/nested/pre-install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+kubectl create namespace "${scenario}-deps"
+helm install -n "${scenario}-deps" spire charts/spire --wait -f $SCRIPT_DIR/deps-values.yaml
+kubectl get all -n "${scenario}-deps"
+kubectl get nodes -o go-template='{{range .items}}{{printf "%s\n" .metadata.uid}}{{end}}' | while read line; do
+	kubectl exec -t spire-server-0 -n "${scenario}-deps" -- spire-server entry create -spiffeID spiffe://example.org/example-cluster/nested-spire -parentID spiffe://example.org/spire/agent/k8s_psat/example-cluster/$line -selector k8s:pod-label:app.kubernetes.io/name:server -downstream
+done
+
+helm install -n "${scenario}" spire-upstream-driver charts/spire --wait -f $SCRIPT_DIR/deps2-values.yaml
```

Comment:
Can we rename deps2 values to better describe what it is used for? 

E.g. upstream-server-values.yaml

Find it quite hard to follow what deps and deps2 represents.

Reply:
I'll rework those bits.